resources.yaml

```diff
@@ -291,21 +273,15 @@ deployment:
     start: 2024-01-28
     end: 2024-02-03
     group: training-ga-embo24
-  training-inte:
-    count: 2
-    flavor: c1.c28m225d50
-    start: 2023-10-19
-    end: 2023-10-20
-    group: training-internat23
-  training-qub-:
+  training-ueb-:
     count: 1
     flavor: c1.c28m225d50
-    start: 2023-10-19
-    end: 2023-10-23
-    group: training-qub-glxy-23
-  training-gm2-:
-    count: 1
+    start: 2023-11-20
+    end: 2023-12-04
+    group: training-ueb-bioinf2023
+  training-plus:
```

Comment:
Is this the one for the R course for 60 students?

Reply:
I think so yes

Reply:
But they can and also should use the normal queue

Reply:
If the resources allocated to this training should also be used for spinning up ITs, then are these (https://github.com/usegalaxy-eu/vgcn-infrastructure/blob/main/resources.yaml#L60-L65) not necessary to be added?

Reply:
That's right, I was too quick to approve, sorry